osx/build.sh

```diff
@@ -37,23 +40,24 @@
 # . http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script
 # . http://stackoverflow.com/questions/2924422/how-do-i-determine-if-a-web-page-exists-with-shell-scripting/20988182#20988182
 # . http://stackoverflow.com/questions/2751227/how-to-download-source-in-zip-format-from-github/18222354#18222354
-#
+# . http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script
+# 
 # 
 # MUST: test the signed .pkg or .app on a target Mac with:
 # . pkgutil --check-signature KA-Lite.pkg -- or;
 # . pkgutil --check-signature KA-Lite.app -- or;
 # . spctl --assess --type install KA-Lite.pkg
 
 
-echo "KA-Lite OS X build script for version 0.16.x and above."
+echo "KA-Lite OS X build script for version 0.17.x and above."
 
 STEP=0
-STEPS=13
+STEPS=15
 
 # TODO(cpauya): get version from `ka-lite/kalite/version.py`
-VERSION="0.16"
-
-PANTRY_CONTENT_URL="http://pantry.learningequality.org/downloads/ka-lite/$VERSION/content"
+VERSION="develop"
```

Comment:
Suggestion: You can assign defaults to variables like so:

    VERSION=${VERSION:-"develop"}

This way, we can set `$VERSION` in the environment, for instance from Bamboo settings.